monitoring_alerts.tf

```diff
@@ -10,6 +10,31 @@ resource "google_monitoring_notification_channel" "slack_alert" {
   }
 }
 
+resource "google_monitoring_uptime_check_config" "http" {
+  display_name = "${var.uptime_check_host}"
+  timeout      = "5s"
+  period       = "60s"
+
```

Comment:
I had to add `project      = "${var.stackdriver_workspace}"` for this to work.

Reply:
Fixed. Mine worked as my project id is the same as my stackdriver workspace.